deploy/template.yaml

```diff
@@ -0,0 +1,1338 @@
+              SSEAlgorithm: AES256
+      Tags:
+        - Key: Name
+          Value: !Sub "${AWS::StackName}-AccessLogs"
+        - Key: Product
+          Value: !Ref ProductTagValue
+        - Key: System
+          Value: !Ref SystemTagValue
+        - Key: Environment
+          Value: !Ref Environment
+        - Key: Owner
+          Value: !Ref OwnerTagValue
+        - Key: Source
+          Value: !Ref SourceTagValue
+        - Key: CheckovRulesToSkip
+          Value: CKV_AWS_18
+
+  AccessLogsBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref AccessLogsBucket
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Sid: AllowLoadBalancerToLogToS3
+            Effect: Allow
+            Principal:
+              AWS: !Sub
+                - "arn:aws:iam::${ElbAccountId}:root"
+                - ElbAccountId: !FindInMap [ ElasticLoadBalancerAccountIds, !Ref AWS::Region, AccountId ]
+            Action:
+              - s3:PutObject
+            Resource: !Sub "${AccessLogsBucket.Arn}/AWSLogs/${AWS::AccountId}/*"
+            Condition:
+              Bool:
+                "aws:SecureTransport": true
+          - Sid: AllowAwsLogDeliveryToLogToS3
+            Effect: Allow
+            Principal:
+              Service: delivery.logs.amazonaws.com
+            Action:
+              - s3:PutObject
+            Resource: !Sub "${AccessLogsBucket.Arn}/AWSLogs/${AWS::AccountId}/*"
+            Condition:
+              StringEquals:
+                "s3:x-amz-acl": bucket-owner-full-control
+                "aws:SourceAccount": !Ref AWS::AccountId
+              Bool:
+                "aws:SecureTransport": true
+          - Sid: AllowAwsLogDeliveryToReadBucketAcl
+            Effect: Allow
+            Principal:
+              Service: delivery.logs.amazonaws.com
+            Action:
+              - s3:GetBucketAcl
+            Resource: !GetAtt AccessLogsBucket.Arn
+            Condition:
+              StringEquals:
+                "aws:SourceAccount": !Ref AWS::AccountId
+              Bool:
+                "aws:SecureTransport": true
+
+  #
+  # CloudFront Distribution
+  #
+
+  CloudFrontDistribution:
+    Type: AWS::CloudFront::Distribution
+    DependsOn: CloudFrontLogsBucket
+    Properties:
+      # checkov:skip=CKV_AWS_86:Ensure Cloudfront distribution has Access Logging enabled
+      DistributionConfig:
+        Aliases:
+          - !FindInMap [EnvironmentVariables, !Ref Environment, BASEURL ]
+        CNAMEs:
+          - !FindInMap [EnvironmentVariables, !Ref Environment, BASEURL ]
+        Comment: !Sub "CloudFront Distribution resource for the ${AWS::StackName} application"
+        DefaultCacheBehavior:
+          AllowedMethods:
+            - GET
+            - HEAD
+            - OPTIONS
+            - PUT
+            - PATCH
+            - POST
+            - DELETE
+          CachedMethods:
+            - GET
```

Comment:
@huwd let's make add something to the team manual about how the frontend is behind a cache. We should probably review the `Cache-Control` headers the app sets too to make sure we're not going to leak any personal data through caching something we shouldn't.

Reply:
Hi @alex9smith - fyi - the CachePolicy is set to not cache anything at the CloudFront dist level.

thanks